scripts/jest/matchers/toWarnDev.js

```diff
@@ -53,7 +54,13 @@ const createMatcherFor = consoleMethod =>
       const isLikelyAComponentStack = message =>
         typeof message === 'string' && message.includes('\n    in ');
 
-      const consoleSpy = (format, ...args) => {
+      const createConsoleSpy = methodName => (format, ...args) => {
+        // Ignore uncaught errors reported by jsdom
+        // and React addendums because they're too noisy.
+        if (methodName === 'error' && shouldIgnoreConsoleError(format, args)) {
```

Comment:
Totally unimportant but I'm just curious– was changing `consoleSpy` to `createConsoleSpy` actually unnecessary? Couldn't `methodName` have just used `consoleMethod`?